### DISCUSSION TITLE
Adguard home using DNS that is not in list

### DISCUSSION THREAD

**Discussion by himazawa** [AdguardTeam/AdGuardHome#6793] [Q&A]
I am using AdGuard Home v0.107.44

I have have set Load Balancing DNS via file. The content of the file is:
 
```
127.0.0.1:5335
1.1.1.1
```

Adguard should only use the DNS servers available in that list but for some reasons I found out that in the last 90 days adguard reported ~10.000 queries to Google DNS.

<img width="1215" alt="image" src="https://github.com/AdguardTeam/AdGuardHome/assets/73994521/4de8909e-026e-40f9-9565-9b062db59943">

The server is running since 2022 but that's the first time I see it querying google dns. I didn't touched the AdGuard config in the last 90 days.

Just in case it's needed:

```
uname -a
Linux DietPi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux
```

**Comment by hagezi** [comment]
@himazawa  Check your configured Fallback DNS servers. `Settings > DNS settings`

![grafik](https://github.com/AdguardTeam/AdGuardHome/assets/104343908/28bce3ab-adc0-41a0-9b27-41b801976b83)

**Reply by himazawa** [reply]
@hagezi the Fallback DNS list is empty

<img width="1154" alt="image" src="https://github.com/AdguardTeam/AdGuardHome/assets/73994521/c1458eeb-b14a-44f2-bd8b-8287e4c6dc23">

**Comment by gordonmilligan** [comment]
I also have this issue. My adguard home's top upstream server is 8.8.8.8:53 and it's also the slowest upstream server. I do not have this in my config. I have no idea where it is getting picked up from.

**Reply by gordonmilligan** [reply]
I solved my issue. I had added in custom DNS to specific clients a long time ago. This was the problem.

I found this by grepping for the offending DNS IP address in the adguard home configuration yaml file.
